sdcm/nemesis.py

```diff
@@ -1570,7 +1570,10 @@ def build_list_of_disruptions_to_execute(self, nemesis_selector=None, nemesis_mu
         Here it kept for future usages and unit testing ability.
         more about nemesis_selector behaviour in sct_config.py
         """
-        nemesis_selector = self.cluster.params.get('nemesis_selector')
+        nemesis_selector = self.cluster.params.get('nemesis_selector', [])
```

Comment:
move this to it's own @property function, so if any other code needs it, it won't forget about adding the `!disabled`

Reply:
fixed in next push.